sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_daily_v2/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+---
+friendly_name: FxA Users Daily
+description: |
+  Usage aggregations per FxA user per day. Build from fxa_users_services_daily.
+  If the user had multiple different entries on a single day, then we grab
+  the attributes from the first service observed that day.
+
+  This lower level model uses the following category of events:
+  event_category IN ('auth', 'auth_bounce', 'content', 'oauth')
```

Comment:
This doesn't seem to match https://github.com/mozilla/bigquery-etl/blob/7b00dd676df06147009050e27d2ea45eba996abf/sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_v2/query.sql#L57-L72

Reply:
Thanks,
I just double checked and I updated the description in the metadata and removed `auth_bounce`. This is fine to do as I checked the `auth_bounce` "event_type" and the only event_type we seem to have in there is `fxa_email - bounced` which is being filtered currently out anyways. So seems safe to not include at all.